docs/didkit/install.md

```diff
@@ -11,15 +11,38 @@ title: Installation
 
 
 
-DIDKit can be installed manually or in containerized form.
+DIDKit can be installed as a package via [cargo](#cargo-install),
+[manually](#manual) from source, or in [containerized](#container) form.
+
+## Cargo install
+
+1. To install the DIDKit command line program on GNU/Linux, MacOS, or
+   Windows+WSL, first install
+   [cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html). 
+2. Install `build-essential` or equivalent tools if they aren't already installed.
```

Comment:
`libssl-dev` is required as well now I believe

Reply:
added